docs/UserGuide.md

```diff
@@ -39,136 +39,281 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
+```
+
+#### Parameters
+| Parameter | Expected Format                              | Explanation                                                |
+|-----------|----------------------------------------------|------------------------------------------------------------|
+| name      | Alphanumeric, capitalized                    | Names are in block letters for clarity and consistency.    |
+| contact   | 8-digit number, starts with 8 or 9           | Ensures the contact number is valid in Singapore.          |
+| email     | Must include "@" and domain, case insensitive| Verifies that the email address is in a standard format.   |
+| job       | Any text, case insensitive                   | Accepts all job titles without case sensitivity.           |
+| income    | Non-negative integers                        | Only positive numbers or zero are valid for income fields. |
+
+#### What to Expect
+- **If Successful:** You'll see a message: "New Contact added. Their ID is `<new customer ID>`."
+- **If There is an Error:** 
+  - Missing name: "Customer requires a name."
+  - Incorrect email format: "Please give a valid email address."
+  - Invalid income entry: "Please use a non-negative number for income."
+
+**Handling Duplicates:**  
+If a customer with the same name, email, job, and income is already saved, you'll get a message: "This customer is already saved as a contact. Their ID is `<new customer ID>`."
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
+---
 
-Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+### Feature 3: Remove Old Customer
 
-### Locating persons by name: `find`
+**Purpose:**  
+This feature allows you to remove records of customers who are no longer using your credit card services.
 
-Finds persons whose names contain any of the given keywords.
+**How to Use It:**  
+- **Command Format:** 
+```
+del <customer id>
+```
+- **Example:** 
+```
+del 69
+```
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+#### Parameters
+| Parameter | Expected Format             | Explanation                                         |
+|-----------|-----------------------------|-----------------------------------------------------|
+| ID        | Integer (0 to the last ID)  | The ID must be a valid integer within the registered range. |
```

Comment:
I think we can clarify that the ID, is of the customer in either 
1. The original list or 
2. Any filtered list after using the `filter` command